schema/composite.gql

```diff
@@ -150,6 +150,11 @@ type CompositeResourceSpec {
   """
   connectionSecret: Secret @goField(forceResolver: true)
 
+  """
+  The `ObjectReference`s for the resources composed by this composite resources.
+  """
+  resourceRefs: [ObjectReference!]!
```

Comment:
@thephred @miloszsobczak does the console need to use this new field?

Reply:
I added this for parity with other ref fields, but console could always use it using fieldPath or json fields